src/DOM/File/Blob.purs

```diff
@@ -0,0 +1,22 @@
+module DOM.File.Blob
+  ( type_
+  , size
+  ) where
+
+import Prelude ((==))
+import Data.Maybe (Maybe(..))
+
+import DOM.File.Types (Blob())
+
+foreign import typeImpl :: Blob -> String
+
+-- | A string indicating the MIME type of the data contained in the `Blob`.
+-- | Returns `Nothing` if the type is unknown.
+type_ :: Blob -> Maybe String
+type_ blob = let blobType = typeImpl blob
+             in if blobType == ""
+                then Nothing
+                else Just blobType
```

Comment:
I also thought that it would be nice to have `MimeType` separated from `affjax`.
I could help with doing that.